eos/observable_TEST.cc

```diff
@@ -127,6 +131,60 @@ class ObservableTest :
                     )")
                 );
             }
+
+            /* Test the names of the kinematic variables against a whitelist */
+            {
```

Comment:
Shouldn't this list be part of another file? It is a bit weird to have it here, no?

Reply:
I followed the instruction in the corresponding issue title. The whitelisting should be done in the test case, since it is a "soft" target. We want users to be able to create their own kinematic variables with arbitrary names; however, the EOS ones should be rather uniform. Put differently, we had agreed not to use a new class like ``KinematicVariableName`` or similar.

Reply:
I agree